src/test/tests.ts

```diff
@@ -896,7 +893,15 @@ describe('Server', function () {
       //do nothing
     });
 
-    client.unsubscribe(subId);
+    setTimeout(() => {
+      // TODO
+      // Hagaico I had the need to add this timeout here
```

Comment:
hmm.. this is interesting, i'm wondering what did we change that brake that test, it sounds broken to start with..